src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.stories.mdx

```diff
@@ -51,9 +51,7 @@ import styles from "./useActiveDescendantListFocus.module.scss"; import {
       }, []);
       const {
           focusedElementProps,
-          visualFocusItemIndex,
           visualFocusItemId,
-          createOnItemClickCallback,
```

Comment:
after refactor we do not need to override the on click function of any item (when user press it by mouse).
this way the code our hook is more simple :)  (I keep the return value for backward compatibility just remove it from docs because its not needed)